anthos-multi-cloud/customize-logs-fluentbit/kubernetes/fluentbit-daemonset.yaml

```diff
@@ -0,0 +1,115 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluentbit-user
+  namespace: logging-system
+  labels:
+    k8s-app: fluentbit-user
+spec:
+  selector:
+    matchLabels:
+      k8s-app: fluentbit-user
+  template:
+    metadata:
+      labels:
+        k8s-app: fluentbit-user
+    spec:
+      nodeSelector:
+        kubernetes.io/os: linux
+      containers:
+      - name: gcp-token-minter
+        image: us.gcr.io/gke-multi-cloud-release/gke-addon-sidecar:gke_multicloud.gke_multicloud_images_20220317_1445_RC00
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: HTTPS_PROXY
+          valueFrom:
+            secretKeyRef:
+              key: HTTPS_PROXY
+              name: proxy-config
+        - name: NO_PROXY
+          valueFrom:
+            secretKeyRef:
+              key: NO_PROXY
+              name: proxy-config
+        command:
+        - /app/cloud/kubernetes/multicloud/addonsidecar/gke_addon_sidecar
+        - run-metadata-server
+        - --http_server=localhost:29681
+        - --ksa_name=user-telemetry-agent
+        - --ksa_namespace=logging-system
+        - --token_audience=${PROJECT_ID}.svc.id.goog
+        - --gcp_project_id=${PROJECT_ID}
+        - --gcp_sts_audience=identitynamespace:${PROJECT_ID}.svc.id.goog://gkemulticloud.googleapis.com/projects/${PROJECT_NUMBER}/locations/${REGION}/${CLUSTER_TYPE}/${CLUSTER_NAME}
```

Comment:
nit: is it possible that environment variables used here aren't defined?

Reply:
These samples are part of a tutorial that will be published on the community page that instructs users to set environment variables: https://github.com/amandawestlake/community/blob/fluentbit-anthosmulticloud/tutorials/customize-anthos-multicloud-logs-with-fluentbit/index.md